src/components/Tokens/TokenDetails/BalanceSummary.tsx

```diff
@@ -61,49 +69,214 @@ const StyledNetworkLabel = styled.div`
+      </BalanceRow>
+    )
+  } else {
+    return (
+      <BalanceRow>
+        <PortfolioLogo currencies={currencies} chainId={chainId} size="2rem" />
+        <BalanceContainer>
+          <BalanceAmountsContainer>
+            <BalanceItem>
+              <ThemedText.SubHeader>
+                {formattedBalance} {tokenSymbol}
+              </ThemedText.SubHeader>
+            </BalanceItem>
+            <BalanceItem>
+              <ThemedText.BodyPrimary>{formattedUsdValue}</ThemedText.BodyPrimary>
+            </BalanceItem>
+          </BalanceAmountsContainer>
+          <StyledNetworkLabel color={color}>{chainName}</StyledNetworkLabel>
+        </BalanceContainer>
+      </BalanceRow>
+    )
+  }
+}
+
+const ConnectedChainBalanceSummary = ({
+  connectedChainBalance,
+  chainId,
+  token,
+  theme,
+}: {
+  connectedChainBalance?: CurrencyAmount<Currency>
+  chainId?: ChainId
+  token: Currency
+  theme: DefaultTheme
+}) => {
+  const hasConnectedChainBalance = chainId && connectedChainBalance && connectedChainBalance.greaterThan(0)
+  if (!hasConnectedChainBalance) return null
+
+  const { label: chainName, color: chainColor } = getChainInfo(asSupportedChain(chainId) ?? ChainId.MAINNET)
+  return (
+    <BalanceSection>
+      <ThemedText.SubHeaderSmall color={theme.neutral1}>
+        <Trans>Your balance on {chainName}</Trans>
+      </ThemedText.SubHeaderSmall>
+      <Balance
+        token={token}
+        chainId={token.chainId}
+        balance={connectedChainBalance}
+        tokenSymbol={token.symbol}
+        color={chainColor}
+        chainName={chainName}
+      />
+    </BalanceSection>
+  )
+}
+
+const PageChainBalanceSummary = ({
+  pageChainBalance,
+  chainId,
+  theme,
+}: {
+  pageChainBalance?: TokenBalance
+  chainId: ChainId
+  theme: DefaultTheme
+}) => {
+  if (!pageChainBalance || !pageChainBalance.token) return null
+  const currency = gqlToCurrency(pageChainBalance.token)
+  return (
+    <BalanceSection>
+      <ThemedText.HeadlineSmall color={theme.neutral1}>
+        <Trans>Your balance</Trans>
+      </ThemedText.HeadlineSmall>
+      <Balance token={currency} chainId={chainId} gqlBalance={pageChainBalance} isInfoTDPEnabled={true} />
+    </BalanceSection>
+  )
+}
 
-  if (!account || !balance) {
+const OtherChainsBalanceSummary = ({
+  otherChainBalances,
+  hasPageChainBalance,
+}: {
+  otherChainBalances?: TokenBalance[]
+  hasPageChainBalance: boolean
+}) => {
+  const navigate = useNavigate()
+  const isInfoExplorePageEnabled = useInfoExplorePageEnabled()
+
+  const hasOtherChainBalances = otherChainBalances && Boolean(otherChainBalances.length)
+  if (!hasOtherChainBalances) return null
+  return (
+    <BalanceSection>
+      {hasPageChainBalance ? (
+        <ThemedText.SubHeaderSmall>
+          <Trans>On other networks</Trans>
+        </ThemedText.SubHeaderSmall>
+      ) : (
+        <ThemedText.HeadlineSmall>
+          <Trans>Balance on other networks</Trans>
```

Comment:
Hmmmm this is a problem unrelated to this PR but good callout... Will fix

Since Circle launched native USDC, all of the previous bridged USDCs should have been renamed to USDC.e / 'Bridged USDC' (you can see the swap component here displaying the correct symbol). So this token on this page is USDC.e, which is different from native USDC

Reply:
try for a different token! i.e. try opening the WETH tdp on another chain